src/vma/event/event_handler_manager.cpp

```diff
@@ -859,6 +862,8 @@ void* event_handler_manager::thread_loop()
 	}
 	BULLSEYE_EXCLUDE_BLOCK_END
 	
+	signal(SIGINT, update_global_exit);
```

Comment:
I have a concern about using signal() and sigaction() in single process. See more at https://www.gnu.org/software/libc/manual/html_node/Signal-and-Sigaction.html.
In addition it can conflict with VMA SIGINT processing https://github.com/Mellanox/libvma/blob/master/src/vma/sock/sock-redirect.cpp#L2485
One more:  Only one signal handler can be installed per signal for the same process. sigaction can return previous that can be used but special code should be implemented to support several signal handlers or the same signal. Take into account that user application can use own signal handler that can conflict with VMA internal.